app/dashboard/views.py

```diff
@@ -706,6 +707,20 @@ def bounty_details(request, ghuser='', ghrepo='', ghissue=0, stdbounties_id=None
                 params['stdbounties_id'] = bounty.standard_bounties_id
                 params['interested_profiles'] = bounty.interested.select_related('profile').all()
                 params['avatar_url'] = bounty.get_avatar_url(True)
+
+
+                snooze_days = int(request.GET.get('snooze'))
```

Comment:
This will 💣 if `snooze` isn't present for whatever reason with a `TypeError`. Can we default to `0`?

Reply:
updated